src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -15,19 +18,23 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniqueNoteList notes;
+    private final QuizQuestionList quiz;
 
     /*
      * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
      * between constructors. See https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
      *
      * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
-     *   among constructors.
+     * among constructors.
      */
+
     {
         notes = new UniqueNoteList();
+        quiz = new QuizQuestionList();
     }
 
-    public AddressBook() {}
+    public AddressBook() {
+    }
```

Comment:
Null constructors should be left out, as they are in the existing classes.

Reply:
Do you want me to remove it or make it as the original form?

Reply:
Eh, never mind. This is too little a detail to squirm over. Merge your pull request.